test/pummel/test-heapsnapshot-near-heap-limit-by-api.js

```diff
@@ -0,0 +1,150 @@
+// Copy from test-heapsnapshot-near-heap-limit.js
+'use strict';
+
+const common = require('../common');
+
+if (common.isPi) {
+  common.skip('Too slow for Raspberry Pi devices');
+}
+
+const tmpdir = require('../common/tmpdir');
+const assert = require('assert');
+const { spawnSync } = require('child_process');
+const fixtures = require('../common/fixtures');
+const fs = require('fs');
+const v8 = require('v8');
+
+const invalidValues = [-1, '', {}, NaN, undefined];
+let errorCount = 0;
+for (let i = 0; i < invalidValues.length; i++) {
+  try {
+    v8.setHeapSnapshotNearHeapLimit(invalidValues[i]);
```

Comment:
I think this can just be `assert.throws(() => v8.setHeapSnapshotNearHeapLimit(invalidValues[i]))`?

Reply:
Done !